common/unsafe/src/main/java/org/apache/spark/unsafe/types/CalendarInterval.java

```diff
@@ -174,12 +191,47 @@ public static CalendarInterval fromDayTimeString(String s) throws IllegalArgumen
         int sign = m.group(1) != null && m.group(1).equals("-") ? -1 : 1;
         long days = m.group(2) == null ? 0 : toLongWithRange("day", m.group(3),
```

Comment:
@dongjoon-hyun  To be consistent, I will also change this. The difference between m.group(2) and m.group(3) is a space.

Reply:
Thank you!

Reply:
I intentionally keep the original to reduce the size of patch.